demisto_sdk/commands/generate_docs/tests/generate_docs_test.py

```diff
@@ -312,6 +312,32 @@ def test_get_input_data_complex():
     assert _value == 'File.Name'
 
 
+def test_generate_playbook(playbook):
+    """Unit test
+    Given
+        - generate_playbook_doc command
+        - playbook
+    When
+        - running the command generate_playbook_doc on playbook in order to generate documentation.
+    Then
+        - Validate That from-version added to the documentation.
+    """
+    from demisto_sdk.commands.generate_docs.generate_playbook_doc import \
+        generate_playbook_doc
+
+    try:
+        playbook.yml.write_dict({'fromversion': '5.0.0'})
+
+        generate_playbook_doc(input=playbook.yml.path)
```

Comment:
You can use the `output` param and give it a tmpdir path instead of trying to remove the file in the end.

Reply:
done